src/tests/gobprepare/importers/test_dump_import.py

```diff
@@ -0,0 +1,100 @@
+import http.client
+from functools import partial
+from io import StringIO
+from unittest import TestCase
+from unittest.mock import MagicMock, patch
+from urllib.error import HTTPError
+
+import pandas as pd
+from pandas import read_csv
+from pandas.errors import ParserError
+from pandas.io.parsers import TextFileReader
+from swiftclient import Connection
+
+from gobcore.datastore.sql import SqlDatastore
+from gobcore.exceptions import GOBException
+from gobprepare.importers.dump_importer import SqlDumpImporter, ObjectDatastore, _load_from_objectstore
+from gobprepare.importers.typing import SqlDumpImporterConfig
+from gobprepare.utils.requests import retry
+
+
+def patch_retry(max_retry, wait, func, self):
+    return retry(max_retry, wait)(partial(func.__wrapped__, self))
+
+
+class TestSqlCsvImporter(TestCase):
+    """Test SqlCsvImporter."""
+
+    def setUp(self) -> None:
+        self.config_objectstore: SqlDumpImporterConfig = {
+            "id": "cfg_objectstore",
+            "depends_on": ["other_cfg"],
```

Comment:
Is het niet mooier om dit soort configuratie uit een json bestand in te lezen i.p.v te hardcoden in code?